index.html

```diff
@@ -1,11 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" href="index.css" />
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Bio Page</title>
 </head>
-<body>
-    
+<body bgcolor="beige">
+    <div align="center">
+        <img id="img" src="image/anu.jpeg">
+        <br>
+        <h2 style="text-align:center;line-height: 5px;">Anuradha Sathiyachalam</h2>
+        <h3 style="text-align:center;line-height: 5px;">Dallas, TX</h3>
+        <h3 style="text-align:center;line-height: 5px;">Email: anoosathyee@gmail.com</h3>
+
+
+
+         <a href="https://www.linkedin.com/in/anuradha-sathiyachalam-9aa49265/"
+                target="_blank">LinkedIn</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="https://github.com/AnooSathyee"
+                target="_blank">GitHub</a>
+        <hr>
```

Comment:
Avoid using content related tags (e.g. the 'hr' tag is used to represent the change of context in an article, story or book) for unrelated purposes.  Consider replacing with a more appropriate element, or removing altogether.